<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Inter</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ventanaClientes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed
{
  background-color: rgb(33, 203, 135, 95);
 color: white;
}

QPushButton
{
   background-color:none;
	 border-radius: .3em;
	border: 1px solid #12ba5b;
}

QPushButton:hover
{
  background-color: #12ba5b;
}

QLineEdit
{
	 border-radius: .3em;
	border: 1px solid  black;
}

QMainWindow{
	background-color: none;
}
</string>
   </property>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Registro de Clientes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDni">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DNI:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNombres">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombres:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblApellidoPaterno">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apellido Paterno:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblApellidoMaterno">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apellido Materno:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDireccion">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dirección:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTelefono">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Teléfono:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDni">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombres">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellidoPaterno">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellidoMaterno">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDireccion">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTelefono">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnListar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Listar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuitar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quitar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistrar">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConsultar">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>180</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModificar">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>220</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>961</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblClientes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>931</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>DNI</string>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombres</string>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>A. paterno</string>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>A. materno</string>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dirección</string>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Celular</string>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnActualizar">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>260</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
